setup/mail-postfix.sh

```diff
@@ -215,6 +215,88 @@ tools/editconf.py /etc/default/postgrey \
 tools/editconf.py /etc/postfix/main.cf \
 	message_size_limit=134217728
 
+# ### MTA-STS - SMTP Mail Transfer Agent Strict Transport Security - SETUP
+# See: https://github.com/mail-in-a-box/mailinabox/pull/1556
+#
+# create the MTA-STS policy; this is used to announce the local MTA-STS
+# policy to senders and the file is served by nginx on mta-sts subdomains at
+# https://example.com/.well-known/mta-sts.txt
+mkdir -p /var/lib/mailinabox/
+cat > /var/lib/mailinabox/mta-sts.txt << EOF
+version: STSv1
+mode: enforce
+mx: $PRIMARY_HOSTNAME
+max_age: 86400
+EOF
+chmod a+r /var/lib/mailinabox/mta-sts.txt
+
+# install the postfix MTA-STS resolver.  the MTA-STS resolver service is
+# used by Postfix to ensure outgoing mail uses TLS when the recipient
+# announces MTA-STS.
+hide_output /usr/bin/pip3 install postfix-mta-sts-resolver
```

Comment:
Should this be altered to include an upgrade parameter if postfix-mta-sts-resolver is already installed?

` hide_output /usr/bin/pip3 install -U postfix-mta-sts-resolver`